README.md

```diff
@@ -1,2 +1,14 @@
 # rome
 A new build system for SugarCRM
+
+# docker
+
+Build the docker image:
+```
+docker build -t registry.sugarcrm.net/rome:1.0.0 .
```

Comment:
lets put it here,

`registry.sugarcrm.net/engineering/rome:latest`

and this is v2.0.0Alpha for now since 1.0 exists in the Mango repo.